cluster/charts/rook-minio-objectstore/values.yaml

```diff
@@ -0,0 +1,45 @@
+# Default values for rook-minio-objectstore.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: IfNotPresent
```

Comment:
This section isn't needed, as there is no version, image option in the CRD (right now).